### PR TITLE
#160617630 Setup SASS processing on the client

### PR DESCRIPTION
**Pull request by kimbugp**
#### What does this PR do?
- Install and setup sass pre-processing on the application to enables the use of both .scss and .css files on the client
#### Description of Task to be completed?
- when you need to use scss to style the react components ,add and import a file with the extension `.scss` or `.sass`
#### How should this be manually tested?
- add styling to the components using sass/scss or css
#### What are the relevant pivotal tracker stories?
- #160617630
#### Screenshots
- below is a html styled using scss

screen shot 2018-10-30 at 21 55 50

[DELIVERS #160617630]

### CODE REVIEW

File: src/css/App.sass [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+$secondary-color: red
+$margin: 16px
+
+.app
+  border-color: $primary-color
+  color: darken($primary-color, 10%)
+  text-align: center
+
+.mission
+  color: darken($secondary-color, 10%)
```
Comment (LehruAngela) [inline src/css/App.sass]: Add new line at end of file

File: src/css/App.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+.app{
+  border-color: $primary-color;
+  color: darken($primary-color, 10%);
+  text-align: center;
+}
+
+.mission{
+  color: darken($secondary-color, 10%);
+}
```
Comment (LehruAngela) [inline src/css/App.scss]: Add new line at end of file here too

File: src/App.js [hunk 3]
```diff
@@ -1,9 +1,12 @@
...
 	render() {
 		return (
-			<div><h1>Authors Haven </h1></div>
+			<div className="app">
+				<h1>Authors Haven </h1>
+				<h2 className='mission'>A Social platform for the creative at heart.</h2>
+			</div>
 		);
 	}
 }
```
Comment (LehruAngela) [inline src/App.js:12]: Let's create a folder called `assets` in the src directory where we will put all the external dependencies. In the assets folder, you can then place the `css`folder you created.

**Review by LehruAngela — CHANGES_REQUESTED** [review]

File: src/css/App.sass (hunk 1 above)
Comment (kimbugp) [reply src/css/App.sass]: noted

File: src/App.js (hunk 3 above)
Comment (kimbugp) [reply src/App.js:12]: Noted

Comment (LehruAngela) [reply src/App.js:12]: You also need to maintain your indentation to 2 spaces, as you have done in the sass files

**Review by LehruAngela — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 3 above)
Comment (kimbugp) [reply src/App.js:12]: @LehruAngela in the eslint config file. We setup indentation using tabs, so whenever I put 2 spaces indentation, it complains. Maybe I should change the Indentation rule!

Comment (LehruAngela) [reply src/App.js:12]: We should remove that indent rule @kimbugp, the indentation looks inconsistent.

Comment (kimbugp) [reply src/App.js:12]: Let me make that change

File: src/assets/css/App.sass [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+$margin: 16px
+
+.app
+  border-color: $primary-color
+  color: darken($primary-color, 10%)
+  text-align: center
+
+.mission
+  color: darken($secondary-color, 10%)
+
```
Comment (LehruAngela) [inline src/assets/css/App.sass]: Add new line here

Comment (kimbugp) [reply src/assets/css/App.sass]: I have added it

**Comment by LehruAngela** [comment]
Please resolve the conflicting files @kimbugp

**Pull request merged**